01_fib/NicholeL/fib.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func abs(args int) (res int) {
+	if args == 0 {
+		return 0
+	}
+	if args > 0 {
```

Comment:
get rid of first three lines and change to `>=`

Reply:
will update